app/src/main/java/com/example/moodtrackerproject/utils/PreferenceManager.kt

```diff
@@ -26,6 +29,16 @@ object PreferenceManager {
     private val valuesMood: ParameterizedType =
         Types.newParameterizedType(List::class.java, MoodModel::class.java)
 
+    private val valuesHealth: ParameterizedType =
+        Types.newParameterizedType(List::class.java, HealthModel::class.java)
+
+    private val valuesTests: ParameterizedType =
+        Types.newParameterizedType(List::class.java, Integer::class.java)
+
+    private val healthJsonAdapter: JsonAdapter<List<HealthModel>> =
+        moshi.adapter<List<HealthModel>>(valuesHealth).nonNull()
```

Comment:
You don't need `List` here: 
`private val moshi = Moshi.Builder().add(KotlinJsonAdapterFactory()).build()`
`private val jsonAdapter: JsonAdapter<HealthModel> = moshi.adapter(HealthModel::class.java)`